### DISCUSSION TITLE
Supabase recommended Remote DB Connection

### DISCUSSION THREAD

**Discussion by magicquin** [supabase/supabase#4315] [Questions]
Hi All,

Hope that everyone is doing well. I have a postgres DB that is connected to another project that i have built. What I am looking at doing is creating new project within Supabase, however there is data one the DB that I would like to incorporate into the new Supabase project as I would like to utilize functionality within Supabase. What I would like to find out is, what would be the best way to do so from a Supabase perspective.

Thank you advance for all any advice and help.

**Comment by soedirgo** [comment]
Hey @magicquin, is this what you're looking for? https://github.com/supabase/supabase/discussions/4168#discussioncomment-1736046
